cpp/daal/include/algorithms/classifier/classifier_training_types.h

```diff
@@ -99,6 +99,7 @@ class DAAL_EXPORT InputIface : public daal::algorithms::Input
 public:
     InputIface(size_t nElements);
     InputIface(const InputIface & other) : daal::algorithms::Input(other) {}
+    InputIface & operator=(const InputIface & other) = default;
```

Comment:
Should we also copy add daal::algorithms::Input(other) as in copy-constructor?

Reply:
Default compiler-generated implementation should do this. No need to implement this manually.

I've unified all this kind of copy constructors and assignment operators in this PR to have definitions in .h and default implementations in .cpp files.